serendipity_hq/aws-ses-monitor-bundle/2.0/config/packages/shq_aws_ses_monitor.yaml

```diff
@@ -0,0 +1,47 @@
+# Aws Ses Monitor
+shq_aws_ses_monitor:
```

Comment:
I suggest less "default config" and rather link the user to the bundle's documentation.

Reply:
@Nyholm , what do you suggest? Should have I to completely remove all the default configuration?

Reply:
Yes please

Reply:
Ok, give me some time... I'll try to fix it by next Tuesday (2019/07/02).

Reply:
No rush. 

Thank you